interfaces/Portalicious/src/locale/messages.xlf

```diff
@@ -1540,6 +1540,15 @@
       <trans-unit id="9022161403115733621" datatype="html">
         <source>We do not share any personal information with other third-parties.</source>
       </trans-unit>
+      <trans-unit id="8433732438274024544" datatype="html">
+        <source>Add note</source>
```

Comment:
I'm surprised this was added, and nothing was removed. We already had this label somewhere. Maybe it's missing an id?